aws-qbusiness-dataaccessor/pom.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>software.amazon.qbusiness.dataaccessor</groupId>
+    <artifactId>aws-qbusiness-dataaccessor-handler</artifactId>
+    <name>aws-qbusiness-dataaccessor-handler</name>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
```

Comment:
This package needs to be a sub module of the root.
See Application as an example: [aws-qbusiness-application/pom.xml#L14](https://github.com/aws-cloudformation/aws-cloudformation-resource-provider-qbusiness/blob/main/aws-qbusiness-application/pom.xml#L14)